src/wasm-linker.cpp

```diff
@@ -366,10 +369,30 @@ bool hasI64ResultOrParam(FunctionType* ft) {
   return false;
 }
 
+void Linker::makeDummyFunction() {
+  bool create = false;
+  // Check if there are indirectly called functions
```

Comment:
This comment should say 'address-taken' functions, since you can compare pointers without calling anything :)